Plugins/TestGeneratorPlugin/TestGeneratorPlugin.swift

```diff
@@ -0,0 +1,23 @@
+import PackagePlugin
+
+/// The Swift Package Manager plugin that generates XCTest cases for annotated .val files as part of
+/// our build process.
+@main
+struct TestGeneratorPlugin: BuildToolPlugin {
+
+  func createBuildCommands(context: PluginContext, target: Target) throws -> [Command] {
+    guard let target = target as? SourceModuleTarget else { return [] }
+    let inputPaths = target.sourceFiles(withSuffix: "val").map(\.path)
+    let outputPath = context.pluginWorkDirectory.appending("ValFileTests.swift")
+
+    let cmd: Command = .buildCommand(
```

Comment:
```suggestion
    let cmd = Command.buildCommand(
```

I really don't feel strongly about it but I think it may be better to avoid using qualification inference when we're just calling a static member directly.

Reply:
Absolutely.